docs/data_formats.md

```diff
@@ -60,19 +58,25 @@ The underlying datatype limits the maximum number of IDs:
 | `uint32`  | 4,294,967,295              |
 | `uint64`  | 18,446,744,073,709,551,615 |
 
-### Tools
-Of course, datasets do not need to be created manually.
-The [webKnossos Cuber](https://github.com/scalableminds/webknossos-cuber) converts image stacks and KNOSSOS cubes into a WKW dataset.
-It also compresses datasets for efficient file storage and creates necessary metadata.
+## Data Formats
 
-## WKW Datasets
-[webknossos-wrap (WKW)](https://github.com/scalableminds/webknossos-wrap) is an optimized data format for 3D voxel image data.
+To bring the above concepts together, webKnossos uses [webknossos-wrap (WKW)](https://github.com/scalableminds/webknossos-wrap) as a container format for volumetric voxel data.
+For sparse skeleton-like structures, webKnossos uses [NML](#NML).
+
+### WKW Datasets
+[webknossos-wrap (WKW)](https://github.com/scalableminds/webknossos-wrap) is a format optimized for large datasets of 3D voxel imagery and supports compression, efficient cutouts, multi-channel and several base datatypes.
 It works well for large datasets and is built with modern file systems in mind.
 Compared to KNOSSOS datasets, it is more efficient because it orders the data within the container for optimal read performance (Morton order).
 WKW is versatile in the image formats it can hold: Grayscale, Multi-Channel, Segmentation, RGB as well as a range of data types (e.g., `uint8`,  `uint16`, `float32`).
 Additionally, WKW supports compression for disk space efficiency.
 
-### WKW Folder Structure
+Each layer of a WKW dataset may contain one of the following:
+* Grayscale data (8 Bit, 16 Bit, Float), also referred to as `color` data
+* RGB data (24 Bit)
+* Segmentation data (8 Bit, 16 Bit, 32 Bit)
+* Multi-channel data (multiple 8 Bit). Note that multi-channel is currently only supported by webKnossos if there are exactly three 8-bit channels.
```

Comment:
```suggestion
```

Reply:
Yeah, fair enough. I thought I'd let it there, since the format supports it and therefore wK might catch up at some point. But it's probably more harm than without it.